cypress/e2e/issue_credential_version_2_use_case.cy.ts

```diff
@@ -75,13 +96,24 @@ describe('Onboarding demo test using issue credential protocol version 2', () =>
 
       const oobId = interception.response?.body.outOfBandRecord.id
 
-      cy.intercept('GET', `${API_URL}/connections?outOfBandId=${oobId}`).as('getConnectionRecord')
-
       cy.request('POST', `${TEST_AGENT_URL}/oob/receive-invitation-url`, body)
 
-      cy.wait(['@getConnectionRecord']).then((inter) => {
-        const record = inter.response?.body[0]
-        cy.wrap(record).its('state').should('not.equal', 'invited')
+      cy.streamRequest<Event>(webSocketConfig, {
```

Comment:
You haven't updated the cy.waits for the other event types yet (credentials and proofs)

Reply:
On it